### ISSUE TITLE
Eliminate snake case naming

### ISSUE THREAD

**Issue by icecrime** [docker/docker#11605]
**Problem**
 
Eliminate all underscore names in docker code
 
**Solution**
 
Replace all underscores variables with camelCase analogues. You can find them with golint tool (`go get -u github.com/golang/lint/golint`).
```
golint ./... | grep -v vendor | grep "don't use underscores in Go names"
```
from project root. Use your common sense when replacing to detect if it's safe.

**Comment by GeorgeMac** [comment]
#dibs

**Comment by reikani** [comment]
#dibs

**Comment by rrarunan** [comment]
@jfrazelle  Needs closing. Fixed in PR #11788 ...
